packages/block-editor/src/hooks/test/generated-variation-class-name.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * WordPress dependencies
+ */
+import { applyFilters } from '@wordpress/hooks';
+import {
+	registerBlockType,
+	unregisterBlockType,
+	registerBlockVariation,
+	unregisterBlockVariation,
+	getBlockType,
+} from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import '../generated-variation-class-name';
+
+describe( 'generated variation className', () => {
+	const defaultBlockSettings = {
+		attributes: {
+			fruit: {
+				type: 'string',
+				default: 'Bananas',
+				source: 'text',
+				selector: 'div',
+			},
+		},
+		save: ( { attributes } ) => (
+			// eslint-disable-next-line react/no-unknown-property
+			<div class={ attributes.className }>{ attributes.fruit }</div>
+		),
+		category: 'text',
+		title: 'block title',
+	};
+
+	const variationBlockSettings = {
+		attributes: {
+			fruit: {
+				type: 'string',
+				default: 'Bananas',
+				source: 'text',
+				selector: 'div',
+			},
```

Comment:
Same as https://github.com/WordPress/gutenberg/pull/61864/files#r1655105267

```suggestion
			fruit: 'Apples',
```